app/Http/Repositories/MailChimpRepository.php

```diff
@@ -307,6 +307,27 @@ public function sendCampaign(string $campaignId) : bool
         }
     }
 
+    /**
+     * @see https://developer.mailchimp.com/documentation/mailchimp/reference/campaigns/#action-post_campaigns_campaign_id_actions_test
+     */
+    public function sendTestNewsletter(string $email, string $campaignId): string {
+      $mailChimpApiArguments = [
+        'test_emails' => ["{$email}"],
+        'send_type' => "html"
+      ];
+      try {
+        $this->post("campaigns/{$campaignId}/actions/test", $mailChimpApiArguments);
+        return "This worked.";
+      } catch(\Exception $e) {
+        $details = [
+          'exception' => $e,
+          'arguments' => $mailChimpApiArguments
+        ];
+        Rollbar::log(Level::WARNING, $details);
+        return false;
```

Comment:
With the return type hints in php, where we have `public function example() : returnTypeHintHere`, everything returned has to return that type. So in the event that `sendTestNewsletter` doesn't succeed, rather than returning false it will throw a PHP error because the method is expecting a string to be returned. If it is expecting a boolean, it'll work.